File: primer_trimestre/poo/php/ejercicios_php/third/libreria/thirdLaw.php
<?php
require_once 'NewtonLaw.php';
class ThirdLaw extends NewtonLaw {
    private float $rForce;

    public function __construct(float $mass, float $acceleration, float $force) {
        parent::__construct($mass, $acceleration, $force);
    }

    public function calculate(): string {
        $force = $this->getForce();
        $this->rForce = -$force;
        return "The reaction force is: " . $this->rForce . " N";
    }
}

File: primer_trimestre/poo/java/proyectos_java/ejercicio_3_java/src/thirdLaw.java
public class thirdLaw extends NewtonLaw {
    private double rForce;
    public thirdLaw(double mass, double acceleration, double force) {
        super(mass, acceleration, force);
    }
    @Override
    public String calculate() {
        double force = getForce();
        double rforce = 0-force;
        return "The reaction force is: " + rforce + " N";
    }
}
